### PR TITLE
translate resource quota.

### PR DESCRIPTION
**Pull request by jianzhangbjz**
As the title shows, I translated the documents of `ReqourceQuota`, welcome review. Thanks. :)
task updated: [k8s.io translation index.xlsx](https://docs.google.com/spreadsheets/d/1FDFCv9RK5nSMgLXhPrJ5k7r5QvHnNEFnXbvoFiM8v20/edit#gid=906253755) line 74.

### CODE REVIEW

File: docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -1,157 +1,101 @@
----
```
Comment (markthink) [inline docs/concepts/policy/resource-quotas.md:1]: 看一下README 文档，一段英文一段中文，格式不正确。

Comment (jianzhangbjz) [reply docs/concepts/policy/resource-quotas.md:1]: updated

File: docs/concepts/policy/resource-quotas.md [hunk 2]
```diff
@@ -1,12 +1,5 @@
...
-assignees:
-- derekwaynecarr
-title: Resource Quotas
```
Comment (markthink) [inline docs/concepts/policy/resource-quotas.md]: 这里不能删除，title 需要翻译，原文也是要用`<!-- --> `包起来

Comment (jianzhangbjz) [reply docs/concepts/policy/resource-quotas.md]: Added it.

File: docs/concepts/policy/resource-quotas.md [hunk 3]
```diff
@@ -6,7 +7,9 @@ redirect_from:
 - "/docs/admin/resourcequota/"
 - "/docs/admin/resourcequota/index.html"
 ---
```
Comment (markthink) [inline docs/concepts/policy/resource-quotas.md:9]: 后面加
```
---
  assignees:
  - derekwaynecarr
 title: 资源配额
 redirect_from:
  - "/docs/admin/resourcequota/"
  - "/docs/admin/resourcequota/index.html"
  ---
```

Comment (jianzhangbjz) [reply docs/concepts/policy/resource-quotas.md:9]: Ok, done.

**Pull request merged**